tests/filecheck/dialects/scf/for_args_types.mlir

```diff
@@ -12,3 +13,17 @@
     "scf.yield"() : () -> ()
   }) : (i32, index, index) -> ()
 }) : () -> ()
+
+// -----
+
+"builtin.module"() ({
+  %lb = "arith.constant"() {"value" = 0 : index} : () -> index
+  %ub = "arith.constant"() {"value" = 42 : index} : () -> index
+  %step = "arith.constant"() {"value" = 7 : index} : () -> index
```

Comment:
Please use the testing dialect for these. I know the test above uses arith.constant as well, but I want to move towards only testing dialect usage, so that test will change in the future as well.

Reply:
I fixed the test above as well ;)